src/controller/java/src/matter/onboardingpayload/OnboardingPayload.kt

```diff
@@ -174,10 +168,6 @@ class OnboardingPayload(
       ")"
   }
 
-  fun addOptionalQRCodeInfo(info: OptionalQRCodeInfo) {
-    optionalQRCodeInfo[info.tag] = info
-  }
```

Comment:
This one is used in QR code generator, but you only implement the Optional support in parser. I think you might need this when you implement the generator part

Reply:
I didn't see where it was being used but I can add this back.

Reply:
Should be resolved by this commit: https://github.com/project-chip/connectedhomeip/pull/31251/commits/1c5223dc36dd125d49e65b3e5a1036e497ce950b